docs/configuration/pki/index.rst

```diff
@@ -248,6 +248,96 @@ certificates used by services on this router.
+the :ref:`http-api`, it will not be enough to simply add the server's 
+certificate and key to the PKI unless that certificate is issued directly by
+a root CA which the client already trusts.
+
+RFC 5246 7.4.2 specifies that during a TLS handshake the server must provide the
+entire chain of public certificates (optionally omitting the root certificate) 
+which authenticate its identity to the client. The first certificate in that 
+list will be the server's certificate, for which the server also has the 
+corresponding private key. Each certificate which follows must directly certify
+the one which precedes it, up to the root certificate. These certifying 
+certificates are ones for which the server will not possess the corresponding 
+private key.
+
+As an example, LetsEncrypt supplies a ``fullchain.pem`` file which contains the 
+entire chain of public certificates concatenated in the correct order, starting 
+with the server certificate and ending with the root certificate. When manually
+adding a LetsEncrypt certificate to Vyos' PKI, such as in the case of a DNS 
+wildcard certificate generated somewhere other than Vyos, the server certificate
+and its private key should be added with the ``set pki certificate`` command and 
+all following certificates should be added with the ``set pki ca`` command 
+without a private key.
+
+The following ``sed`` command will break a concatenated file of certificates 
+into several single-line base 64 representations which can be directly used in 
+the ``set pki certificate <name> certificate`` and 
+``set pki ca <name> certificate`` commands.
+
+.. code-block:: bash
+
+  sed -r ':a;N;$!ba;s/\n//g;s/-{5}[A-Z ]+-{5}/ /g;s/(^\s+|\s+$)//g;s/\s+/\n\n/g' fullchain.pem
+
+  # This can also be set as a bash alias with some escaping
+  alias pem-vyos="sed -r ':a;N;\$!ba;s/\\n//g;s/-{5}[A-Z ]+-{5}/ /g;s/(^\\s+|\\s+$)//g;s/\s+/\\n\\n/g'"
+  pem-vyos fullchain.pem
+
+For example, assume you are trying to use an externally generated LetsEncrypt 
+DNS wildcard certificate into Vyos' PKI for use with the :ref:`http-api`. You 
+have four files from LetsEncrypt:
+
+* ``cert.pem`` - contains the wildcard certificate for your server
+* ``privkey.pem`` - contains the private key which matches ``cert.pem``
+* ``chain.pem`` - contains the public certs for LetsEncrypt and ISRG (root CA)
+* ``fullchain.pem`` - concatenation of ``cert.pem`` plus ``chain.pem``
+
+Running the ``sed`` command above on ``chain.pem`` or ``fullchain.pem`` will 
+output two or three (respectively) single line base 64 certificates which can 
+be directly copied and pasted. The last certificate in the output will be the 
+root CA, and the second-to-last will be LetsEncrypt's. 
+
+.. code-block:: none
+
+  set pki ca isrg certificate <ISRG's certificate data>
+  set pki ca lets_encrypt certificate <LetsEncrypt's certificate data>
+
+Then running the ``sed`` command on ``cert.pem`` and ``privkey.pem`` will output
```

Comment:
its not documented yet, but you can use
`import pki certificate my_cert file cert.pem`
`import pki certificate my_cert key-file privkey.pem`
or in config mode
`run import pki certificate my_cert file cert.pem`
`run import pki certificate my_cert key-file privkey.pem`

its also work with the fullchain.pem file, but not for the ca